pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -180,4 +180,11 @@
      * @return message redelivery count
      */
     int getRedeliveryCount();
+
+    /**
+     * Get schema version of the message.
+     * @since 2.3.1
```

Comment:
I would be using 2.4.0, unless it will be ported to branch-2.3 and include in 2.3.1

Reply:
Changed to 2.4.0